includes/admin.inc

```diff
@@ -408,6 +408,12 @@ function islandora_solr_admin_settings($form, &$form_state) {
     '#description' => t('Add navigation params to object links.'),
     '#default_value' => variable_get('islandora_solr_search_navigation', FALSE),
   );
+  $form['islandora_solr_tabs']['default_display_settings']['islandora_solr_search_field_value_separator'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Field value separator'),
+    '#description' => t('Characters to separate values in multivalued fields.'),
+    '#default_value' => variable_get('islandora_solr_search_field_value_separator', ''),
```

Comment:
Related to the below comments about having 2 different displays run by a single choice, by default you don't show either of those defaults here.

Also, if we allow HTML to be added here and added to the display later. Should we be running a filter_xss or something on it?